avc/sps.go

```diff
@@ -370,7 +370,9 @@ func (a *SPS) ConstraintFlags() byte {
 
 // GetSARfromIDC - get Sample Aspect Ratio from IDC index
 func GetSARfromIDC(index uint) (uint, uint, error) {
-	if index < 1 || index > 16 {
+	if index == 0 { // index 0 is a special case; don’t emit an error.
```

Comment:
According to Section E.2.1 of 14496-10_2020 the value 0 is unspecified, so I don't see that a specific value should be returned.  On the other hand, 0,0 as aspect ratio is toxic.

Anyway, I think this is OK, but I'd like the comment to say that the value is unspecified in the spec, and that we choose 1,1, as a good default value.